pkg/controller/masteruserrecord/masteruserrecord_controller.go

```diff
@@ -83,7 +85,7 @@ func (r *ReconcileMasterUserRecord) Reconcile(request reconcile.Request) (reconc
 
 	// Fetch the MasterUserRecord instance
 	instance := &toolchainv1alpha1.MasterUserRecord{}
-	err := r.client.Get(context.TODO(), request.NamespacedName, instance)
+	err := r.client.Get(context.TODO(), types.NamespacedName{Namespace: config.GetOperatorNamespace(), Name: request.Name}, instance)
```

Comment:
In case you have cluster_scope_resource (primary/secondary), the namespace value will be empty for reconcile_callback for cluster_scope_resource.  To make sure we have namespace for such thing .. we have get namespace from env here too [code_link](https://github.com/codeready-toolchain/member-operator/blob/master/pkg/controller/useraccount/useraccount_controller.go#L99).

Reply:
I know, but these CRs are all namespace scoped, aren't they...?

Reply:
I don't recall why this code was changed, perhaps it was in response to a failing test however I've modified it back to use the `NamespacedName` and the tests still seem to pass, so I'll leave it like that.

Reply:
ok, just don't introduce any changes in the file when it's not needed, so we can have the PR smaller :-).